### PR TITLE
Add coverage testing target

### PR DESCRIPTION
**Pull request by Mrokkk**

### CODE REVIEW

**Comment by szborows** [comment]
this ain't working. need to knock more :)

**Comment by Mrokkk** [comment]
Why the log is empty, is this some Travis bug?

**Comment by szborows** [comment]
Plz send me your mobile privately. Texting will be a little bit better to sync ;)

**Comment by szborows** [comment]
BTW, regarding empty logs - they started to appear after one of my commits, namely "gcovr dependency added"

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -5,3 +5,14 @@ set(SOURCES
...
 target_link_libraries(tests yebash)
+
+add_custom_target(tests-run DEPENDS tests COMMAND ./tests)
+
+if (COVERAGE)
+    add_custom_target(clean-coverage COMMAND find ${CMAKE_BINARY_DIR} -name '*.gcda' -exec rm {} \\\; )
+    add_custom_target(tests-cov
+        DEPENDS clean-coverage tests-run
+        COMMAND gcovr -r ${CMAKE_SOURCE_DIR}/src ${CMAKE_BINARY_DIR})
+        # COMMAND gcovr --html --html-details -o coverage.html -r ${CMAKE_SOURCE_DIR}/src ${CMAKE_BINARY_DIR})
```
Comment (szborows) [inline test/CMakeLists.txt]: what about this?

Comment (Mrokkk) [inline test/CMakeLists.txt]: `gcovr: error: no such option: --html`, said the Travis ;)

**Pull request merged**